### ISSUE TITLE
Allow both JSON and Text Logging

### ISSUE THREAD

**Issue by mostlyjason** [ajbrown/angular-loggly-logger#4]
Hey AJ one of our developers recently spent some time to improve the unit tests.  We submitted a pull request for you to review.  

Also, we wanted to get your opinion on logging JSON and string objects.  Loggly only supports one type per field. So if we log a string to json.message then we can't also log a JSON object there. This will prevent the JSON fields from showing in the field explorer and other places.  One solution we thought of was to log strings to json.message and log objects to json.messageObj.  This way you can send either type of data and both will show in Loggly correctly.  Do you think this solution would work for your library?  If so, we can submit a pull request for it.

Thanks,
Jason

**Comment by ajbrown** [comment]
Jason,

Thanks for the message!  I saw the pull request come in yesterday, I just wanted to verify the tests on my end before I accepted it.  I'll get to that this evening and pull it in.  I'll also get Travis CI hooked up correctly now that the tests actually work right.

I think the solution you proposed sounds great.  Go ahead and submit the pull request and I'll get it in right away.

It's awesome to see that this library is not only useful to someone, but that there's interest in making it better.  I *really* appreciate the feedback and collaboration.  It's been sitting idle for a while as I've been focused on other things, so the improvements are definitely welcomed.

Thanks!
